pkgs/development/python-modules/qmk-dotty-dict/default.nix

```diff
@@ -0,0 +1,22 @@
+{ buildPythonPackage, fetchPypi, lib, pytest, setuptools-scm }:
+
+buildPythonPackage rec {
+  pname = "qmk_dotty_dict";
+  version = "1.3.0.post1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-O2EeOTZgv6poNcaOlHhLroD+B7hJCXi17KsDoNL8fqI=";
+  };
+
+  nativeBuildInputs = [ setuptools-scm ];
+
+  doCheck = false;
```

Comment:
Why are tests disabled? Why is there no pythonImportsCheck?

Reply:
There is a single test that pass on macOS but not on Linux, I haven't found a way to fix it

Reply:
It's disabled for `dotty-dict` package too. (for the same reason, I guess)

Reply:
But dotty-dict has a pythonImportsCheck?

Reply:
There is none. https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/dotty-dict/default.nix